Auth0.podspec

```diff
@@ -1,12 +1,12 @@
 version = `agvtool mvers -terse1`.strip
 
 web_auth_files = [
-  'Auth0/A0ChallengeGenerator.h',
-  'Auth0/A0ChallengeGenerator.m',
-  'Auth0/A0RSA.h',
-  'Auth0/A0RSA.m',
-  'Auth0/A0SHA.h',
-  'Auth0/A0SHA.m',
+  'Auth0/ObjectiveC/A0ChallengeGenerator.h',
+  'Auth0/ObjectiveC/A0ChallengeGenerator.m',
+  'Auth0/ObjectiveC/A0RSA.h',
+  'Auth0/ObjectiveC/A0RSA.m',
+  'Auth0/ObjectiveC/A0SHA.h',
+  'Auth0/ObjectiveC/A0SHA.m',
```

Comment:
The ObjectiveC files were moved into a folder, to be able to create a SPM target with them.